qemu/tests/timedrift_with_stop.py

```diff
@@ -91,12 +112,32 @@ def run(test, params, env):
             host_delta = ht1_ - ht0_
             guest_delta = gt1_ - gt0_
             drift = abs(host_delta - guest_delta)
+
+            # Get hardware time after current iteration
+            if not stop_with_signal:
+                if params.get("os_type") == "linux" and rtc_clock == "host":
+                    (hhwt1_, ghwt1_) = get_hw_time()
+                    host_hw_delta = float(hhwt1_) - float(hhwt0_)
+                    guest_hw_delta = float(ghwt1_) - float(ghwt0_)
+
             # kvm guests CLOCK_MONOTONIC not count when guest is paused,
             # so drift need to subtract stop_time.
             if not stop_with_signal:
                 drift = abs(drift - stop_time)
                 if params.get("os_type") == "windows" and rtc_clock == "host":
                     drift = abs(host_delta - guest_delta)
+                if params.get("os_type") == "linux" and rtc_clock == "host":
+                    error.context("Check the hardware time on guest and host")
```

Comment:
error_context.context

Reply:
@yanan-fu thanks. Updated.
Please help to review again.